wetmap/src/components/newModals/shopModal/index.tsx

```diff
@@ -1,75 +1,41 @@
 import React, { useState, useContext, useEffect, useRef } from 'react';
-import Itinerary from '../../itineraries/itinerary';
 import { itineraries } from '../../../supabaseCalls/itinerarySupabaseCalls';
+import { updateDiveShop } from "../../supabaseCalls/shopsSupabaseCalls";
```

Comment:
Hey Andrew, I grabbed the needed query for you out of the mobile version and have imported it into the index for you its called "updateDiveShop" it will handle the needed pieces for updating a dive shop's name, bio or photo link

Reply:
Thanks for this!